index.js

```diff
@@ -155,15 +155,22 @@ class CoverageSonar extends CoverageBase {
     }
 
     /**
-     * Return links to the Sonar badge and project
-     * @method getLinks
-     * @param   {Object} jobId    Project ID
-     * @return  {Promise}         An object with coverage badge link and project link
+     * Return links to the Sonar project and coverage metadata
+     * @method getInfo
+     * @param   {Object}  config
+     * @param   {String}  config.jobId      Screwdriver job ID
+     * @param   {String}  config.startTime  Job start time
+     * @param   {String}  config.endTime    Job end time
+     * @return  {Promise}                   An object with coverage badge link and project link
      */
-    _getLinks(jobId) {
+    _getInfo({ jobId, startTime, endTime }) {
+        // eslint-disable-next-line max-len
+        const coverageUrl = `${this.config.sonarHost}/api/measures/search_history?component=job:${encodeURIComponent(jobId)}&metrics=coverage&from=${encodeURIComponent(startTime)}&to=${encodeURIComponent(endTime)}`;
```

Comment:
I think we should make the call to this `coverageUrl ` and parse the result here. Just give the number back.